paramz/parameterized.py

```diff
@@ -29,7 +29,11 @@
 #===============================================================================
 
 import numpy; np = numpy
-from re import compile, _pattern_type
+from re import compile
+try:
+    from re import _pattern_type
+except ImportError:
+    from re import Pattern
```

Comment:
This should be `from re import Pattern as _pattern_type`, right? Or, alternatively, the whole try-catch block could go away and the library could do the same thing that `re` was doing in 3.6 and before: `_pattern_type = type(compile("", 0))`.

Reply:
Indeed, missed `as _pattern_type`. The second option is not obvious (though of course works).